vector/src/main/java/im/vector/app/features/notifications/NotifiableMessageEvent.kt

```diff
@@ -45,4 +47,7 @@ data class NotifiableMessageEvent(
     val type: String = EventType.MESSAGE
     val description: String = body ?: ""
     val title: String = senderName ?: ""
+
+    val imageUri: Uri?
```

Comment:
Does it worth adding `@Transient` here?

Reply:
Android-studio says "This annotation is not applicable to target 'member property without backing field or delegate'", I guess it's not necessary here

Reply:
OK, thanks for checking!